releasetasks/templates/firefox/bb_update_verify.yml.tmpl

```diff
@@ -1,7 +1,7 @@
 {% for channel in release_channels %}
 {% set uv_totalchunks = 6 %}
 {% for chunk in range(1, uv_totalchunks + 1) %}
-{% set uv_buildername = "release-{}_{}_update_verify_{}_{}".format(branch, platform, channel, chunk) %}
+{% set uv_buildername = "release-{}_{}_{}_update_verify".format(branch, product, platform) %}
 -
     taskId: "{{ stableSlugId(uv_buildername) }}"
```

Comment:
probably need to augment more to pass releasetasks tests, since we're getting a conflicting taskId for each iteration of this loop when you don't include chunk in the stableSlugId string....